spec/spec-helper.coffee

```diff
@@ -1 +1,11 @@
 require 'coffee-cache'
+
+exports.expectMappings = (currentLayer, upperLayer, mappings) ->
```

Comment:
Maybe these parameters should just be named `layerA` and `layerB` or `targetLayer` and `sourceLayer`?